ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/FilterResourceDropdown.tsx

```diff
@@ -47,12 +47,13 @@ function FilterResourceDropdown({
     return (
         <Select
             variant="single"
-            aria-label="resource"
+            toggleAriaLabel="resource filter menu toggle"
+            aria-label="resource filter menu items"
             onToggle={onToggle}
             onSelect={onResourceSelect}
             selections={resource}
             isOpen={isOpen}
-            className="pf-u-flex-none"
+            className="pf-u-flex-basis-0"
```

Comment:
Apparently I didn't test this well enough last week, as `none` breaks the layout...